mews-operations/availabilityBlock.md

```diff
@@ -41,7 +41,8 @@
 
 ### Request
 
-[PlatformAddress]/api/channelManager/v1/processAvailabilityBlock
+* **Test environment:** `TODO`
+* **Production environment:** `TODO`
```

Comment:
@patrikmiksovsky95 I am gonna need your help here. Can anyone else do this? Maybe @Vaclav-Hanselka? I don't know how nor I have access to PCI afaik.